app/views/state_file/questions/nj_eligibility_health_insurance/edit.html.erb

```diff
@@ -9,7 +9,7 @@
       <div class="form-group">
         <%= f.cfa_radio_set(
               :eligibility_all_members_health_insurance,
-              label_text: t(".label", filing_year: @filing_year),
+              label_text: t(".label_html", filing_year: @filing_year),
```

Comment:
[sand] very minor but it looks like `app/controllers/state_file/questions/nj_eligibility_health_insurance_controller.rb:6` is another instance of looking up the year in the config instead of using `current_tax_year`